ChangeLog.d/configuration-split.txt

```diff
@@ -0,0 +1,18 @@
+Changes
+   * Cryptography and platform configuration options have been migrated
+     from the Mbed TLS library configuration file mbedtls_config.h to
+     crypto_config.h is the TF-PSA-Crypto configuration file,
+     see Mbed-TLS/mbedtls/docs/proposed/config-split.md
```

Comment:
I still think we should not mention `config-split.md`. It's an architectural document, not a user-facing document. And it isn't even in this product (but even in Mbed TLS, I don't think we should reference it in the changelog: it's for the wrong audience).